CESC-submission/paper.tex

```diff
@@ -62,15 +62,15 @@
 \end{abstract}
 
 
 \section{Introduction}
 Usually, when obtaining digital services, we deal with traditional systems that are centrally managed. Most of the time, we resort to third party providers, e.g., commercial companies,  to obtain services like file storage, content distribution, computation outsourcing, and many others. Despite being effective and widely deployed, this centrally-managed paradigm introduces several trust, cost, and transparency issues. It requires establishing complex business relationships with these companies, where customers usually overprovision their needs in order to handle future peak demands~\cite{Kassa13,Nicolae14}. Also, it constrains the customers with the service specifications these companies can offer, like the geographic coverage and service speed, not to mention the limited visibility into the real status of the system regarding its performance and the available amount of resources.
 
 
-These issues motivated the community to revisit old ideas of peer-to-peer (P2P) based models in which anyone is allowed to join the system and serve others. In order to encourage the collaborative work and compliance with the protocol, payments are provided in return, which creates a market for trading resources. This paradigm builds flexible systems, where anyone can start or stop the service at any time. It also scales more easily with demand and extends the network coverage because peers from anywhere can join. Furthermore, this paradigm builds transparent and equitable ecosystems in which participants can negotiate service terms and price directly instead of having a few entities monopolizing the market. 
+These issues motivated the community to revisit old ideas of peer-to-peer (P2P) based models in which anyone is allowed to join the system and serve others. In order to encourage the collaborative work and compliance with the protocol, payments are provided in return, which creates a market for trading resources. This paradigm builds flexible systems, where customers can start or stop the service at any time. It also scales more easily with demand and extends the network coverage because peers from anywhere can join. Furthermore, this paradigm builds transparent and equitable ecosystems in which participants can negotiate service terms and price directly instead of having a few entities monopolizing the market.
```

Comment:
anyone -> customers

not sure if service-providers can jump in and out whenever they want in most service marketplaces :)

Reply:
I see your point, I will edit.